### PR TITLE
Add depoy to Heroku button

### LINKED ISSUE #69: Add deploy to Heroku button
Add a button in README.md for automatically deploying the project to Heroku. This is useful for testing the project in production. 

This is a temporary fix till we get automatically deployng with Travis working again.

### PR DESCRIPTION
**Pull request by ameistad**
This add a button to README.md for automatically deploying the project to Heroku.
app.json contains instructions for auto deploying.
runtime.txt specifies the python version for Heroku to use.

Closes #69

### CODE REVIEW

**Comment by SanketDG** [comment]
The commit message should say `Closes #69`

File: app.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Djenesis contributr",
+    "description": "Find projects to contribute to",
+    "logo": "https://i.imgur.com/odDO1Fx.png",
```
Comment (SanketDG) [inline app.json]: Do we really need this logo?

Comment (nitinpaultifr) [inline app.json]: Seems irrelevant.
